airflow_tecton/operators/tecton_feature_table_job_operator.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2022 Tecton, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pandas
+from typing import Any, Callable, Collection, Mapping, Sequence, Union
+
+from airflow.models import BaseOperator
+from airflow.utils.context import Context
+
+from airflow_tecton.hooks.tecton_hook import TectonHook
+from airflow_tecton.operators.df_utils import ingest_feature_table_with_pandas_df
+from airflow_tecton.operators.job_utils import wait_until_completion, kill_job
+
+
+class TectonFeatureTableJobOperator(BaseOperator):
+    """
+    An Airflow operator that launches a Tecton Feature Table ingestion job,
+    and waits for its completion. If the latest job with the same params
+    is running, it will cancel it first. If the latest job with the same
+    params is successful, this will return without running a new job.
+
+    Use this if you want to submit a Tecton job via Airflow and control
```

Comment:
hmm i forgot about this bit. you might need to modify the MDS code to handle this, because before we passed in a flag to disable retries